src/request/sign-transaction/SignTransaction.css

```diff
@@ -1,154 +1,300 @@
+#confirm-transaction {
+    position: relative;
+}
+
 #layout-container {
+    overflow: hidden;
+}
+
+#layout-container.standard .hide-standard,
+#layout-container.checkout .hide-checkout {
+    display: none;
+}
+
+#layout-container .nq-card-footer {
+    border: 0;
+    padding: 3rem 4rem;
+}
+
+#layout-container .nq-card-body {
+    padding: 2rem 4rem 0rem;
+
+}
+
+#layout-container > #account-details {
+    position: absolute;
+    overflow: hidden;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    border-radius: 1rem;
+    background-color: rgba(255,255,255, 0.75);
+    z-index: -1;
+    opacity: 0;
+    padding: 4rem;
     display: flex;
     flex-direction: column;
-    width: 100%;
+    align-items: center;
+    transition: opacity .4s, z-index .4s;
 }
 
-/*
-#######################################
-#### S T A N D A R D   L A Y O U T ####
-#######################################
-*/
+#layout-container.open > #account-details {
+    z-index: 59;
+    opacity: 1;
+}
 
-.transaction .accounts {
-    flex-direction: row;
-    align-items: flex-start;
+#layout-container #effect-container {
+    overflow: hidden;
+    border-radius: 1rem;
 }
 
-.transaction .account {
+#layout-container > #effect-container > * {
+    transition: filter .4s, opacity .4s;
+    -webkit-filter: blur(0px);
+    -moz-filter: blur(0px);
+    -o-filter: blur(0px);
+    -ms-filter: blur(0px);
+    filter: blur(0px);
+    opacity: 1;
+}
+
+#layout-container.open > #effect-container > .page-header,
+#layout-container.open > #effect-container > .page-body {
+    -webkit-filter: blur(20px);
+    -moz-filter: blur(20px);
+    -o-filter: blur(20px);
+    -ms-filter: blur(20px);
+    filter: blur(20px);
```

Comment:
Should we use `rem` here too, instead of `px`?

Reply:
so, what is the general consensus? I would say that for a blur effect px should suffice.